Resources/Prototypes/_LostParadise/Catalog/Fills/Lockers/suit_storage.yml

```
@@ -0,0 +1,71 @@
+  - type: StorageFill
+    contents:
+        - id: AirTankFilled
+        - id: ClothingOuterHardsuitMercenary
+        - id: ClothingMaskGasMercenary
+        - id: JetpackMiniFilled
+        - id: ClothingShoesBootsMagMercenaryFilled
+  - type: AccessReader
+    access: [["Captain"], ["Mercenary"]]
+
+- type: entity
+  parent: [ SuitStorageBase, StorageFillEVASuitEngineer ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)